packages/hs-react-ui/src/components/Text/Text.tsx

```diff
@@ -43,11 +42,11 @@ const Progress = styled.div`
   ${({ size }: { size: string }) => css`
     background: linear-gradient(
         45deg,
-        rgba(255, 255, 255, 0.2),
-        rgba(0, 0, 0, 0.2),
-        rgba(255, 255, 255, 0.2),
-        rgba(0, 0, 0, 0.2),
-        rgba(255, 255, 255, 0.2)
+        rgba(255, 255, 255, 0.75),
+        rgba(0, 0, 0, 0.75),
+        rgba(255, 255, 255, 0.75),
+        rgba(0, 0, 0, 0.75),
+        rgba(255, 255, 255, 0.75)
```

Comment:
what was the reason for this change?

Reply:
I was trying to use the Text component inside of the button rather than reimplementing the loading style, but the opacity made the loading bar look really bad on some background colors because it bled through and seriously degraded the quality of the progress bar.